src/components/human-impact-layers/human-impact-layers-component.jsx

```diff
@@ -9,9 +9,9 @@ import { HUMAN_PRESSURE_LAYER_ID } from 'constants/human-pressures';
 
 const HumanImpactLayers = ({ map, rasters, setRasters, setLayerVisibility, activeLayers }) => {
   const humanImpactLayerActive = activeLayers.find(l => l.id === HUMAN_PRESSURE_LAYER_ID);
-  const alreadyChecked = humanImpactLayerActive && humanPressuresLandUse.reduce((acc, option) => ({ 
+  const alreadyChecked = humanImpactLayerActive && (humanPressuresLandUse.reduce((acc, option) => ({
     ...acc, [option.value]: rasters[option.value]
-  }), {}) || {};
+  }), {}) || {});
```

Comment:
It's more a question than request, but does this last `|| {});` do anything? In case when `humanImpactLayerActive` will be false, we will return nothing instead of empty object, is that right?

Reply:
Maybe it would be cleaner with a ternary, but it is working the same way.
Just added the parenthesis for disambiguation

Reply:
:ok_hand: